flip/templates/study/_outcome_form.html

```diff
@@ -20,22 +20,6 @@
 </div> {# form-group #}
 
 <div class="form-group">
-  <div class="row">
-    <div class="col-md-7">
-      {{ form.content_topic.label_tag }}
-      {% if form.content_topic.field.required %}
-         <span class="text-danger">*</span>
-      {% endif %}
-      {% render_field form.content_topic class='form-control' %}
-      {% if form.content_topic.errors %}
-        <p class="text-danger">
-          <small>{{ form.content_topic.errors.0 }}</small></p>
-      {% endif %}
-    </div> {# col-md-7 #}
-  </div>
-</div> {# form-group #}
-
-<div class="form-group">
```

Comment:
According to the structure described by Marie, the content_topic shouldn't be there anymore.

Please check #23938.